recipes/xtl/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+from conans import ConanFile, tools
+
+
+class XtlConan(ConanFile):
+    name = "xtl"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xtensor-stack/xtl"
+    description = "The x template library"
+    topics = ("conan", "templates", "containers", "algorithms")
```

Comment:
```suggestion
    topics = ("conan", "templates", "containers", "algorithms")
    settings = "os", "compiler", "arch", "build_type"
```

Reply:
Why excluding package_id instead? We don't need setting in this recipe, it's only copying the files to the package folder, no cmake interaction.

Reply:
either one approach or the one you say would be fine

Reply:
"excluding package_id" makes sense to me.